scripts/helpers.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+#######################################
+# Tell the user what programs to install for a specific task.
+# Arguments:
+#   Name of the program or actual command, a string.
+#   External link for easier installation help.
+# Outputs:
+#   Prints the required program name and the external link (if given).
+#######################################
+function program_required() {
+  if [ ! -x "$(command -v ${1})" ]; then
+    echo "${1} is not installed in the computer..."
+    if [ "${2}" ]; then
+      echo "Check out this link: ${2}"
+    fi
+    exit 1
+  fi
+}
+
+#######################################
+# Check for the PIP program.
+# Arguments:
+#   None
+# Outputs:
+#   None
+#######################################
+function pip_required() {
+  program_required "pip" "https://pip.pypa.io/en/stable/installing/"
+}
+
+#######################################
+# Check for the NPM program.
+# Arguments:
+#   None
+# Outputs:
+#   None
+#######################################
+function npm_required() {
+  program_required "npm" "https://nodejs.org/en/download/"
+}
+
+#######################################
+# Check for the CircleCI program.
+# Arguments:
+#   None
+# Outputs:
+#   None
+#######################################
+function circleci_required() {
+  program_required "circleci" "https://circleci.com/docs/1.0/local-cli/"
+}
+
+#######################################
+# Check for the PNPM program.
+# Arguments:
+#   None
+# Outputs:
+#   None
+#######################################
+function pnpm_required() {
+  program_required "pnpm" "https://pnpm.js.org/en/latest/installation/"
```

Comment:
We currently support pnpm 6, so let's use this URL: https://pnpm.io/6.x/installation